photon-ml/src/integTest/scala/com/linkedin/photon/ml/algorithm/RandomEffectCoordinateTest.scala

```diff
@@ -55,17 +58,17 @@ class RandomEffectCoordinateTest extends SparkTestUtils with GameTestUtils {
   @Test(dataProvider = "numEntitiesDataProvider")
   def testScore(numEntities: Int): Unit = sparkTest("testScore") {
     val (coordinate, model) = generateRandomEffectCoordinateAndModel(
-      randomEffectId, featureShardId, numEntities, NumTrainingSamples, Dimensionality)
+      randomEffectId, featureShardId, numEntities, NUM_TRAINING_SAMPLES, DIMENSIONALITY)
 
     val score = coordinate.score(model)
 
-    assertEquals(score.scores.count, numEntities * NumTrainingSamples)
+    assertEquals(score.scores.count, numEntities * NUM_TRAINING_SAMPLES)
     assertTrue(score.scores.map(_._2).collect.forall(_ == 0.0))
   }
 }
 
 object RandomEffectCoordinateTest {
-  val NumTrainingSamples = 1000
-  val Dimensionality = 10
-  val NumPartitions = 4
+  val NUM_TRAINING_SAMPLES = 1000
+  val DIMENSIONALITY = 10
+  val NUM_PARTITIONS = 4
```

Comment:
I know it's inconsistent with how we've done things here, but the Scala style guide actually recommends upper camel case for constants:

http://docs.scala-lang.org/style/naming-conventions.html#constants-values-variable-and-methods

I've been trying to nudge us in that direction, but we should probably make a wholesale change, or just have them all use the Java convention.